descriptors/ModuleDescriptor-template.json

```diff
@@ -294,6 +294,42 @@
         }
       ]
     },
+    {
+      "id": "acquisitions-unit-assignments",
```

Comment:
The same interface name is going to be for invoicing and finance modules. According to [Okapi Reference - Multiple interfaces](https://github.com/folio-org/okapi/blob/master/doc/guide.md#multiple-interfaces):
> Normally, Okapi proxy allows exactly one module at once to provide a given interface. By using `interfaceType` `multiple` in the `provides` section, Okapi allows any number of modules to implement the same interface. The consequence, however, is that the user of the interface must choose which module to call by specifying HTTP header `X-Okapi-Module-Id`

If we would like to implement the same interface for acq modules, `interfaceType": "multiple"` should be specified and UI (or any other client) has to send `X-Okapi-Module-Id` (value should include module name with version e.g. `mod-orders-6.0.0`). Also in this case I believe the endpoint can be renamed to `/acquisitions-unit-assignments` and all the modules will provide the same endpoints.
Alternative option is to rename interface name to something like `orders.acquisitions-unit-assignments`

Reply:
`orders.acquisitions-unit-assignments` please.... likewise, we'll have `invoices.acquisitions-unit-assignemnts`, etc.